content/_data/changelogs/weekly.yml

```
@@ -25371,11 +25371,53 @@
   - version: '2.487'
     date: 2024-11-27
     changes:
-    - type: security
-      message: Important security fix.
-      references:
-        - url: /security/advisory/2024-11-27/
-          title: security advisory
+      - type: security
+        message: Important security fix.
+        references:
+          - url: /security/advisory/2024-11-27/
+            title: security advisory
+
+  - version: '2.488'
+    date: 2024-12-02
+    changes:
+      - type: bug
+        category: regression
+        pull: 10000
+        issue: 73907
+        authors:
+          - ridemountainpig
+          - daniel-beck
+        pr_title: '[JENKINS-73907] Fix double-escaped tooltips in "Help for feature"'
+        message: |-
+          Fix doubleescaped tooltips in "Help for feature".
```

Insert "-" that is removed by the changelog generator.
```suggestion
          Fix double-escaped tooltips in "Help for feature".
```